microsoft-365/compliance/create-and-manage-inactive-mailboxes.md

```diff
@@ -23,6 +23,9 @@ ms.custom:
 description: "Create and manage inactive mailboxes that retain the contents of deleted mailboxes in Microsoft 365."
 ---
 
+
+> [!IMPORTANT]
+> Removing a Exchange license from a user account does not override any retention policies or holds in place on the mailbox. However, it places the mailbox in a tombstone state, rather than an inactive state. The content cannot be accessed by eDiscovery Tools or Content Searches until a license is reapplied to the user account. CAUTION If the user account is deleted while the mailbox is in a tombstone state, all access to the mailbox content will be lost.
```

Comment:
```suggestion
> Removing an Exchange license from a user account does not override any retention policies or holds in place on the mailbox. However, it places the mailbox in a tombstoned state, rather than an inactive state. The content cannot be accessed by eDiscovery tools or content searches until a license is reapplied to the user account.

> [!CAUTION]
> If the user account is deleted while the mailbox is in a tombstoned state, all access to the mailbox content will be lost.
```